scripts/release/auto_merge_pr.py

```diff
@@ -0,0 +1,59 @@
+#!/usr/bin/env python
+
+# --------------------------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for license information.
+# --------------------------------------------------------------------------------------------
+
+import json
+import logging
+import requests
+import shlex
+import subprocess
+import sys
+
+logger = logging.getLogger(__name__)
+logger.setLevel(logging.DEBUG)
+ch = logging.StreamHandler()
+ch.setLevel(logging.DEBUG)
+logger.addHandler(ch)
+
+
+def auto_merge(headers, number):
+    # curl -X PUT https://api.github.com/repos/OWNER/REPO/pulls/PULL_NUMBER/merge
+    merge_url = f'https://api.github.com/repos/Azure/azure-cli-extensions/pulls/{number}/merge'
+    logger.debug(merge_url)
+    # merge_method: merge, squash or rebase
+    # we use `sqush and merge` button
+    body = {
+        'merge_method': 'squash'
+    }
+    try:
+        r = requests.put(merge_url, json=body, headers=headers)
+    except requests.RequestException as e:
+        raise e
+    if r.status_code != 200:
+        logger.debug(r)
+        logger.debug(r.text)
+        sys.exit(1)
+
+
+def main():
+    try:
+        # get azclibot token
+        with open('/tmp/token.txt') as f:
+            token = f.readline().replace("\n", "")
```

Comment:
Why not fetch token from keyvault?

Reply:
In old steps, we can use environment variable `Azure subscription` in Azure CLI option.
![image](https://user-images.githubusercontent.com/18628534/172520543-0e786e5c-e2ef-4c3e-b8ca-eee3cc248592.png)
So it is easy to use azure cli command.

But new step I use python script which not support environment variable.
![image](https://user-images.githubusercontent.com/18628534/172520601-3942a875-e700-47f0-9f8e-a04b06eedb52.png)
And after the job finished (about 4 mins) this agent will be delete, so I think it is safe.

Reply:
Can't we use **Arguments**?